packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -7,6 +7,7 @@ import { ThemeProvider, lightTheme } from '@strapi/design-system';
 import { useTracking, useAppInfo, TrackingProvider } from '@strapi/helper-plugin';
 import { Router } from 'react-router-dom';
 import { createMemoryHistory } from 'history';
+import 'whatwg-fetch';
```

Comment:
Why do we need this?

Reply:
Oh, this https://github.com/jsdom/jsdom/issues/1724#issuecomment-720727999

Then it should be put in the `admin-test-utils` not in each test file imo :)

Reply:
basically we had a problem with the way we handle unit tests here https://github.com/strapi/strapi/blob/main/packages/core/admin/admin/src/pages/MarketplacePage/tests/server.js before the rest.get calls worked with the axios call but we discovered the new fetch implementation inside the useFetchMarkeplaceProviders and useFetchMarketplacePlugins we discovered could be solved with this pollyfill whatwg-fetch of the fetch. If you have better solutions are welcome

Reply:
nah i think it's the right solution, but we should just put it in the correct setup file in `admin-test-utils` with a note.